src/main/components/urlBar.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+
+const UrlBar = (props) => {
+  const method = (props.AorB === 'B' ? 'POST' : 'GET')
```

Comment:
Can we call this prop something clearer than `AorB`—maybe `defaultMethod`? Also may be more intuitive to store `method` in state (instead of selected, since in some cases the user won’t actually be making a selection)